src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -135,7 +135,7 @@ public EventSource(Configuration configuration)
             if (_backOff.GetReconnectAttemptCount() > 0 && _retryDelay > TimeSpan.FromMilliseconds(0))
             {
                 TimeSpan sleepTime = _backOff.GetNextBackOff();
-                _logger.LogInformation("Waiting " + sleepTime.TotalMilliseconds + " milliseconds before reconnecting...");
+                _logger.LogInformation("Waiting {0} milliseconds before reconnecting...", sleepTime.TotalMilliseconds);
```

Comment:
Similar to above, you don't want to do the string concatenation unless the Information log level is actually enabled.